tests/driver_dsp0401/main.c

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the DSP0401 4 digit display.
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <inttypes.h>
+
+#include "dsp0401_params.h"
+#include "dsp0401.h"
+#include "xtimer.h"
+
+#define TEST_DELAY      (2U)               /* 2 seconds delay between each test */
+#define WORD_DELAY      (750U * US_PER_MS) /* 750 milliseconds delay between each word */
+#define SCROLL_DELAY    (200U)             /* 200 milliseconds delay between character shift */
```

Comment:
disagree, looking more closely, this value is correct. The delay is done with `xtimer_usleep` in main.c. The scroll function internal converts to microseconds.
Maybe we should defnie a `xtimer_msleep` function ?